docs/data/material/components/app-bar/app-bar.md

```diff
@@ -154,7 +154,7 @@ You can override this behavior by setting the `enableColorOnDark` prop to `true`
 
 {{"demo": "EnableColorOnDarkAppBar.js", "bg": true}}
 
-## Experimental APIs
+## Toolpad
 
 ### DashboardLayout
 
```

Comment:
Why do we have `"hideToolbar": true` on this demo? Can we remove this?

x the other demos